### DISCUSSION TITLE
posts/2023/12/field-essay-11/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#32] [General]
# posts/2023/12/field-essay-11/

https://guozheng.netlify.app/posts/2023/12/field-essay-11/

**Comment by liechi** [comment]
菠萝蜜好吃吗？我去海南开会时见过。其实初见时很想尝尝，但遇到的当地人都不怎么推崇菠萝蜜，建议试试榴莲。一尝之下，觉得榴莲也不过如此，对菠萝蜜就更没什么期待，待了一个多周都没有买。

**Reply by residualsun1** [reply]
我觉得还好，但是不会有想吃的冲动，属于“你剥了放那儿，我要是嘴闲或许会吃两片”那种，倒是榴莲我就会比较想吃哈哈。

然后，菠萝蜜的甜一般都有点粘稠腻喉，不过也要看品种，有一些甜而不腻的我也会比较喜欢吃。这晚掰的我就吃了很多片。
